wordpress-mobile/WordPress-Android
# [Gutenberg] media upload progress reattach

#### Description
This PR introduces upload progress reattachment so the progress updates are reflected on the UI when you exit and enter the Editor again.

#### Related PRs
- [gutenberg-mobile PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/505)
- [gutenberg PR](https://github.com/WordPress/gutenberg/pull/13516)

![reattachprogress](https://user-images.githubusercontent.com/6597771/51778321-d7184a80-20df-11e9-9e35-2f2c0dedd43e.gif)


#### To test:
1. start a new draft
2. insert an image
3. once it starts uploading, exit the editor
4. open the Post in the editor again
5. observe the upload progress bar is updated correspondingly.

#### Note:
@marecar3 the base branch corresponds to PR #9120, which also needs to be updated with latest `develop` to clear the commit history difference seen here.

Update release notes:
